### ISSUE TITLE
Is Selawik no longer updated since 2015? Or you guys might abandon this?

### ISSUE THREAD

**Issue by ToddIndry37** [Microsoft/Selawik#15]
Hi Microsoft! Are you _probably_ forgot about Selawik, the "open source" replacement for Segoe UI typeface? This repository never been updated since 2015 (the initial year release of this font), and there's a lot of changes year-to-year.

There's something need to change for this whole typeface:
1. Italic styles
2. More weights like Segoe UI, I guess...?
3. Add several style sets (or something)
4. Font variable support, just like Segoe UI Variable on Windows 11
5. Redesigned Selawik, like Windows 11 version of Segoe UI

We'll wait if you probably might working on redesigned Selawik or something in the works.

**Comment by Twilightssuperb** [comment]
6. Cyrillic glyphs support is welcome too...

**Comment by ian-maurmann** [comment]
There's been no reply from Microsoft in a month on this, and no response to other tickets either.

**Comment by ToddIndry37** [comment]
> There's been no reply from Microsoft in a month on this, and no response to other tickets either.

@ian-maurmann Yeah, I don't know why Microsoft abandon Selawik (the open source version of Segoe UI) for very long time now. Hopefully they'll update it somehow to match the Variable one.

**Comment by ian-maurmann** [comment]
It's been another month now. Might be time to fork.

(I don't think I have the skill to match the new Win11 variable one though)

**Comment by alysdexia** [comment]
> > There's been no reply from Microsoft in a month on this, and no response to other tickets either.
> 
> [@ian-maurmann](https://github.com/ian-maurmann) Yeah, I don't know why Microsoft abandon Selawik (the open source version of Segoe UI) for very long time now. Hopefully they'll update it somehow to match the Variable one.

Update your brain to English.  As opposed to a long nontime.
